edk2toollib/utility_functions.py

```diff
@@ -177,9 +177,11 @@ def RunCmd(cmd, parameters, capture=True, workingdir=None, outfile=None, outstre
     endtime = datetime.datetime.now()
     delta = endtime - starttime
     endtime_str = "{0[0]:02}:{0[1]:02}".format(divmod(delta.seconds, 60))
+    returncode_str = "{0:#010x}".format(c.returncode)
```

Comment:
You can also just do returncode_str = hex(c.returncode) I believe.

Reply:
Either way I think is fine

Reply:
Thanks for looking at this.  The reason I chose .format() was to limit the size of the converted returncode so that the dashes would all line up nicely in predictable fashion (I assumed a 32 bit returncode).